aws/SDK_CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 <!-- Do not manually edit this file, use `update-changelogs` -->
+0.10.0 (April 13th, 2022)
+=========================
+**Breaking Changes:**
+- ⚠ ([smithy-rs#1318](https://github.com/awslabs/smithy-rs/issues/1318)) We updated the MSRV from 1.56.1 to 1.58.1 in the last release but failed to mention it in the changelog.
+
+
```

Comment:
We should just merge the two `0.10.0` sections, and also demote the 2021 edition change from breaking changes since it's not a breaking change. We don't need to say we forgot for the SDK since we didn't release the SDK.